### ISSUE TITLE
Is UserScatteredDataParallel necessary for Synchronized-BatchNorm?

### ISSUE THREAD

**Issue by HuShaohanAI** [CSAILVision/semantic-segmentation-pytorch#150]
is it right that only use SynchronizedBatchNorm2d, DataParallel, patch_replication_callback?

**Comment by hangzhaomit** [comment]
No, they are two separate features.
